### ISSUE TITLE
Bootstrap 4

### ISSUE THREAD

**Issue by melanieceraso** [nickberens360/atomic-docs#48]
Do you have any direction on how we can swap out Bootstrap 3 to use Bootstrap 4 with AtomicDocs?  Thanks.

**Comment by nickberens360** [comment]
@melanieceraso the tabs in atomic docs are dependent on bootstrap so we would need to re-write that portion in order to remove BS3. Have you tried adding BS4 to what's already there? I assume all sorts of bad things happen lol

**Comment by melanieceraso** [comment]
Yeah, it's not pretty.  I have found a basic work-around for now... so it's getting the job done.  Just wondered if there was a better way to do it.  lol.  Thanks.

**Comment by nickberens360** [comment]
I'll take a look at pulling out bootstrap 3 and going with a different tab solution. ill let you know what i come up with. might not be till the weekend though

**Comment by melanieceraso** [comment]
That would be amazing.  I really love using it... and it seems now that people are starting to move into bootstrap 4 a bit... and I decided to finally make the jump.  

I just commented out the bootstrap css references within the site.css and then put a link to the bootstrap 4 cdn in the page header and that seemed to do the trick.  It doesn't want to play nice when I like out to the v4 js but... at least I can get the css to play well.  

And no rush, man... enjoy the nice long weekend... it's good to get away from the keyboard from time to time.  ;)

**Comment by melanieceraso** [comment]
FWIW, I did some SASS to fix up the tabs after I had put in Bootstrap 4.   It wasn't actually thrown off that much in terms of styling and the functionality of them still works.   I added the following to the site.scss file under the atomic-core:

`#atomsWrap .codeBlocks .nav-tabs-atomic {
  padding-left: 0;
  margin-bottom: 0;
  li {
    list-style-type: none;
    display: inline-block;
    margin: 0;
    padding: 10px 15px 15px;
  }
}

#atomsWrap .copyBtn {
  top: -41px;
}`

**Comment by nickberens360** [comment]
Awesome @melanieceraso thanks for digging into this!

**Comment by melanieceraso** [comment]
You're welcome!  :)

<http://www.apruve.com/>


*Melanie A. Ceraso*
Web Application Designer, *Apruve*
*p* 612.812.8173
*w* <http://www.apruve.com/> www.apruve.com
